Core Experience rechartering (#2780)
PostHog/posthog.com: contents/handbook/people/team-structure/core-experience.md

[See team structure page](/handbook/people/team-structure/team-structure)

> 

Reviewer: I do think we need a more concrete team mission? If you were to describe to a new team member, what does Core Experience do in 5 seconds, what would we say?

Author: I'd use the F-word...

Reviewer 2: I like this as a mission: https://github.com/PostHog/posthog.com/pull/2780#discussion_r787981843

## Mission

Company goal that resonates most with this team:

> Quality: Our core features (insights, recordings, feature-flags / experimentation) work like a Swiss watch.

Team Core Experience aims to:

> - Increase the quality of our features across all dimensions (see scope below).

Reviewer 3: This came up as a distinction between core experience and core analytics - core experience's contributions to user experience are often at a system level and span across features. Said another way, core experience is responsible for making sure all features work well together as a cohesive product. Thoughts?

Reviewer 2: > "core experience's contributions to user experience are often at a system level and span across features. Said another way, core experience is responsible for making sure all features work well together as a cohesive product. "

That's a nice framing.

Feels like you could use it to test if you're getting off track too. As a hypothetical example: "We've spent a lot of time ${working on the specifics of how paths works} recently. We should focus on ${helping people get to and from paths} instead"

Reviewer: Definitely think this helps clarify things, 👍 to the general idea!

- Maintain a coherent strategy for frontend and API development (technologies, tools and methodology).

> - Ensuring our features are used the way we expect them to.

Reviewer: ```suggestion
- Ensure our features are used easily (don't make me think!) and solve users' needs. 
```

> - Understanding users and their needs... and solving for that.

Reviewer: ```suggestion
- Understand different the different types of users and their needs... and solving for that.
```

## Goals

What is quality and how do we track it?

We track specific goals/targets depending on feature. For example:

- Qualitative data (what users are saying)
- Measured data (KPIs, performance metrics, etc)
- Internal company feedback (meeting company goals)

> Main metric: number of discoveries.

Reviewer: ```suggestion
Main metric: number of [Discoveries](/handbook/product/metrics#what-is-a-discovery).
```

Reviewer 2: Do we list other metrics?

@clarkus suggested user feedback. So could have a metric of number of user interviews? As a proxy for "user feedback gathered"

@rcmarron suggested "number of bugs" (IIRC). Could be gamed but there's something there. Something between number of sentry errors, number of issues opened, number of issues closed, speed of first response to issue, speed to close issue. Trying to think of two(ish) measures that have tension between them. So you can't just make it so users can't raise issues and then celebrate zero issues raised :)

Also quantitative measures -> page speed budget?

Reviewer: Would love if we could incorporate another set of metrics, particularly to measure UX/perceived quality, however I'm not sure how we can measure this effectively in practice. Having a metric that's difficult to measure an influence seems worse than having no metric (e.g. # of interviews seems quite hard to improve by sheer effort of the Engineering Team)

## Scope

> We handle everything not owned by Core Analytics and Platform.

Reviewer 2: ```suggestion
### In Scope

* work that advances the quality of contributor experience
* work that advances the quality of user experience

### Out of Scope

* work towards the company's Deployment goal
```

Reviewer: Building up on @pauldambra's suggestion,

```suggestion
### In scope

- Work that ensures a consistent and high quality user experience across the entire app (including user experience of features not built by this team).
- Work that enables enterprise and high scale organizations to use PostHog.
- Work that ensures non-technical users and people who didn't set up PostHog can use it to drive insights successfully.
- Development and maintenance of core features aimed at qualitative analysis (e.g. recordings).
- Work towards a high quality experience on secondary interfaces (API, emails, support channels).
- Work that advances the quality of contributor experience.

### Out of scope

- Query functionality and data analysis functionality.
- Event ingestion and processing.
- Data quality, data processing and any feature whose objective is to aid in building data analysis (e.g. cohorts, actions). 

```

Reviewer 3: > Work that ensures non-technical users and people who didn't set up PostHog can use it to drive insights successfully.

While I agree with this point it feels limiting. We should work to ensure **all users**, regardless of technical level or previous experience, can use posthog to answer their questions.

Reviewer: While I think we ultimately want all users to be able to use PostHog, focusing on everyone is the same as not focusing on anyone. Even from very recent calls, it's very clear that we solve really well for technical people and not at all for non-technical, I think we should deliberately focus (at least for now) on improving for the latter.

> We handle everything people in the team have worked on.

Reviewer 3: This is basically saying we own all the stuff we built or touched recently. I think there's some snark in here, but it does bring up the issue of who owns what after it's built.

> We handle everything people in the team have worked on.

Reviewer: Hmm not 100% sure about this. For instance, I could see the case of CoreAn taking over the querying part of recordings, or CoreExp taking over the interaction of Paths/Funnels with Recordings. While generally this would be true, I wouldn't word it as a hard rule. Maybe as below? But still unsure it adds additional value.

```suggestion
- We own (maintain and improve) features we've built, unless there's a total/partial hand-off to another team?
```
